src/main/java/nl/thehyve/whereabouts/domains/Instance.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2019 The Hyve
+ * This file is distributed under the MIT License (see accompanying file LICENSE).
+ */
+
+package nl.thehyve.whereabouts.domains;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Data
+@Entity
+public class Instance {
+    private @Id @GeneratedValue(strategy = GenerationType.IDENTITY) Long id;
+    private String address;
+    private String sourceQuery;
+
+    Instance() {}
```

Comment:
Instead of writing these constructors explicitly, you can add `@NoArgsConstructor @RequiredArgsConstructor` to the class and define the fields as `private @NonNull String address` and private @NonNull String sourceQuery.
Then Lombok does the work for you.